docs/usage/bot-comparison.md

```diff
@@ -0,0 +1,83 @@
+# Bot comparison
```

Comment:
One more entry worth mentioning here is the license model. 

Renovate uses `GNU Affero General Public License.` 

Dependabot uses `Prosperity Public License 2.0.0`